### ISSUE TITLE
Detect severity from Detekt reports

### ISSUE THREAD

**Issue by Urmil-Ally** [NFesquet/danger-kotlin_detekt#8]
Detekt has [different reporting levels](https://arturbosch.github.io/detekt/pages/kdoc/detekt-api/io.gitlab.arturbosch.detekt.api/-severity/index.html) but this plugin detects only 3 of them. Is there a way to report back exactly what the library reports?

**Comment by Urmil-Ally** [comment]
I would like to know if there is a simpler way to override the severity of existing rules in detekt. 
I am aware of writing custom rules but I am looking for an easy way to do so via yml file where one can activate or deactivate rules with one line. 

For example: 
```
style
  ClassOrdering:
```
This is reported as `Style` severity but I would like to override it to `Warning` or `error` for it to work with Danger.js

Any help is appreciated. 

@NFesquet any insights into this.
